data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2030,10 +2072,38 @@
 #endif
 #endif
 
-        [message]
-            speaker=unit
-            message= _ "Ack! What are those things!"
-        [/message]
+        [if]
+            [variable]
+                name=unit.id
+                equals="Abhai"
+            [/variable]
+            [then]
+                [message]
+                    speaker=Abhai
+                    message= _ "Ah... I had forgotten about this beast."
```

Comment:
If you have a unit with high enough mp you can move straight to Abhai's home and avoid the event. Then when you move Abhai he can trigger it. I did this on my first playthrough of this mission.

Reply:
Hmm - another option would be to use an `[event]name=enter_hex` or `[event]name=exit_hex` with `[cancel_action]` to make the unit stop when they enter the water. That would have the advantage of showing the tentacles first, instead of letting the unit get stuck in the village with the tentacles attacking them - although with Abhai around I guess the tentacles aren't a threat.

Reply:
For a quick resolution, I suggest just not adding an extra string here, and turning my previous comment into a new Github issue to improve later.

Reply:
#9295

Reply:
Removed Abhai's line.